### PR TITLE
Install mage outside module

### LINKED ISSUE #1591: go get: installing executables with 'go get' in module mode is deprecated.
This ticket is regarding the discussion for outdated method of installing mage.

When inside the hugo repo, issuing `go get github.com/magefile/mage` throws this.

```
$ go get github.com/magefile/mage
go: downloading github.com/magefile/mage v1.11.0
go get: installing executables with 'go get' in module mode is deprecated.
        To adjust and download dependencies of the current module, use 'go get -d'.
        To install using requirements of the current module, use 'go install'.
        To install ignoring the current module, use 'go install' with a version,
        like 'go install example.com/cmd@latest'.
        For more information, see https://golang.org/doc/go-get-install-deprecation
        or run 'go help get' or 'go help install'.
```

@bep, if you want, I can submit a PR for the same.

### PR DESCRIPTION
**Pull request by santosh**
Fixes #1591

### CODE REVIEW

**Comment by santosh** [comment]
@bep, not sure which branch to create pull request against. I have chosen master.

File: content/en/contribute/development.md [hunk 1]
```diff
@@ -154,9 +154,11 @@ go install
 Hugo relies on [mage](https://github.com/magefile/mage) for some convenient build and test targets. If you don't already have it, get it:
 
 ```
-go get github.com/magefile/mage
+go get github.com/magefile/mage@latest
 ```
 
+This will install mage executable in `$GOPATH` without affecting current project’s module.
```
Comment (bep) [inline content/en/contribute/development.md:160]: Remove this line. It's not (at least not in general) correct, and it's outside the scope of this change.

Comment (santosh) [reply content/en/contribute/development.md]: Updated.

File: content/en/contribute/development.md [hunk 2]
```diff
@@ -154,7 +154,7 @@ go install
 Hugo relies on [mage](https://github.com/magefile/mage) for some convenient build and test targets. If you don't already have it, get it:
 
 ```
-go get github.com/magefile/mage
+go get github.com/magefile/mage@latest
```
Comment (jmooring) [inline content/en/contribute/development.md:157]:
```suggestion
go install github.com/magefile/mage@latest
```

Comment (santosh) [reply content/en/contribute/development.md:157]: @jmooring, okay, so what is the difference here and why does `go get` works for me in this case?

Comment (jmooring) [reply content/en/contribute/development.md:157]:
```bash
go version
```

> go version go1.17.4 linux/amd64

```bash
rm $(which mage)
go get github.com/magefile/mage@latest
```

> go get: installing executables with 'go get' in module mode is deprecated.
	To adjust and download dependencies of the current module, use 'go get -d'.
	To install using requirements of the current module, use 'go install'.
	To install ignoring the current module, use 'go install' with a version,
	like 'go install example.com/cmd@latest'.
	For more information, see https://golang.org/doc/go-get-install-deprecation
	or run 'go help get' or 'go help install'.

```bash
which mage
```

> /home/jmooring/go/bin/mage

**Pull request merged**